src/app/navbar/navbar.component.ts

```diff
@@ -1,14 +1,15 @@
 import { Component, OnInit } from '@angular/core';
-
+import {Clock} from '../clock.model';
+declare var require: any;
 @Component({
   selector: 'app-navbar',
   templateUrl: './navbar.component.html',
   styleUrls: ['./navbar.component.css']
 })
 export class NavbarComponent implements OnInit {
-
-  constructor() { }
-
+  $logo;
```

Comment:
Dlaczego znów pole zawiera $? (co dziwne - dla odmiany tym razem na początku) Czy to jakaś naleciałość z PHP? Nie spotkałam się dotychczas z takim sposobem definiowania zmiennych w Angularze.